spec/libraries/aws_engine_spec.rb

```diff
@@ -68,18 +73,15 @@ def initialize
           @contents.push(obj)
         end
 
-        def contents
-          @contents
-        end
+        attr_reader :contents
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'f' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)